dll/cpl/access/lang/cs-CZ.rc

```diff
@@ -109,155 +117,155 @@ END
           -1, 12, 20, 222, 29
-    AUTOCHECKBOX "&Use shortcut", IDC_CONTRAST_ACTIVATE_CHECK, 12, 49, 222, 14
-    GROUPBOX "High Contrast Color Scheme", -1, 6, 83, 234, 44
-    LTEXT "Current Color Scheme:", -1, 12, 94, 222, 11
+    AUTOCHECKBOX "Po&užít klávesovou zkratku", IDC_CONTRAST_ACTIVATE_CHECK, 12, 49, 222, 14
+    GROUPBOX "Barevné schéma Vysokého kontrastu", -1, 6, 83, 234, 44
+    LTEXT "Aktuální barevné schéma:", -1, 12, 94, 222, 11
     COMBOBOX IDC_CONTRAST_COMBO, 12, 103, 222, 56,
              CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_VISIBLE | WS_TABSTOP
     DEFPUSHBUTTON "OK", IDOK, 135, 207, 50, 14
-    PUSHBUTTON "Cancel", IDCANCEL, 189, 207, 50, 14
+    PUSHBUTTON "Storno", IDCANCEL, 189, 207, 50, 14
 END
 
 IDD_MOUSEKEYSOPTIONS DIALOGEX 0, 0, 246, 220
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Mouse Keys Settings"
+CAPTION "Nastavení Myš klávesnicí"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Keyboard shortcut", -1, 6, 11, 234, 62
-    LTEXT "The shortcut for Mouse Keys is:\nPress <Left Alt+Left Shift+Num Lock>.",
+    GROUPBOX "Klávesová zkratka", -1, 6, 11, 234, 62
+    LTEXT "Klávesová zkratka pro Myš klávesnicí je:\nStistkněte <Levý Alt+Levý Shift+Num Lock>.",
           -1, 12, 20, 222, 29
-    AUTOCHECKBOX "&Use shortcut", IDC_MOUSEKEYS_ACTIVATE_CHECK,
+    AUTOCHECKBOX "Po&užít klávesovou zkratku", IDC_MOUSEKEYS_ACTIVATE_CHECK,
                  12, 49, 222, 14
-    GROUPBOX "Pointer speed", -1, 6, 83, 234, 83
-    LTEXT "&Top speed:", -1, 12, 96, 48, 11
-    LTEXT "Low", -1, 66, 96, 18, 11
+    GROUPBOX "Rychlost kurzoru", -1, 6, 83, 234, 83
+    LTEXT "&Nejvyšší:", -1, 12, 96, 48, 11
+    LTEXT "Nízká", -1, 66, 96, 18, 11
     CONTROL "", IDC_MOUSEKEYS_SPEED_TRACK, "msctls_trackbar32", TBS_AUTOTICKS | WS_TABSTOP,
             90, 94, 114, 20
-    LTEXT "High", -1, 204, 96, 18, 11
-    LTEXT "&Acceleration:", -1, 12, 123, 48, 11
-    LTEXT "Slow", -1, 66, 123, 18, 11
+    LTEXT "Vysoká", -1, 204, 96, 24, 11
+    LTEXT "&Zrychlení:", -1, 12, 123, 48, 11
+    LTEXT "Pomalé", -1, 66, 123, 22, 11
     CONTROL "", IDC_MOUSEKEYS_ACCEL_TRACK, "msctls_trackbar32", TBS_AUTOTICKS | WS_TABSTOP,
             90, 121, 114, 20
-    LTEXT "Fast", -1, 204, 123, 18, 11
-    AUTOCHECKBOX "&Hold down Ctrl to speed up and Shift to slow down", IDC_MOUSEKEYS_SPEED_CHECK,
+    LTEXT "Rychlé", -1, 204, 123, 20, 11
+    AUTOCHECKBOX "&Podržet Ctrl pro zrychlení a Shift pro zpomalení", IDC_MOUSEKEYS_SPEED_CHECK,
                  12, 142, 222, 14
-    LTEXT "Use MouseKeys when NumLock is:", -1, 6, 171, 120, 11
-    AUTORADIOBUTTON "O&n", IDC_MOUSEKEYS_ON_RADIO, 162, 171, 30, 11
-    AUTORADIOBUTTON "Of&f", IDC_MOUSEKEYS_OFF_RADIO, 198, 171, 30, 11
-    AUTOCHECKBOX "&Show MouseKey status on screen", IDC_MOUSEKEYS_STATUS_CHECK,
+    LTEXT "Použít Myš klávesnicí když NumLock je:", -1, 6, 171, 120, 11
+    AUTORADIOBUTTON "Z&ap.", IDC_MOUSEKEYS_ON_RADIO, 162, 171, 30, 11
```

Comment:
Is the extra point because you abbreviated the actual word?

Reply:
Yes.